sway/scene_descriptor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdlib.h>
+#include <sway/scene_descriptor.h>
+
+static void descriptor_destroy(struct sway_scene_descriptor *desc) {
```

Comment:
I just set it up that way because that seems to be the style across the rest of sway. Also, according to contributing.md (at least for wlroots) all destroy functions must accept NULL.